app/_kong_plugins/ai-proxy-advanced/examples/sdk-usupported-model.yaml

```
@@ -0,0 +1,56 @@
+title: 'OpenAI SDK: Use an unsupported LLM model'
+description: |
+  {{site.base_gateway}} can attempt to support models that aren’t pre-configured with format transformers or are untested.
+
+  {:.warning}
+  > **Caution**: The following use cases are unsupported but may work depending on your setup. Use at your own discretion.
+
+  When setting up for unsupported models, you must configure the `route_type` to preserve mode. This approach ensures that the request and response are passed through without any transformations.
+
+  For example, using the Whisper-2 audio transcription model, using the below configuration for the Whisper-2 transcription model, you can `POST` a file for transcription, using `multipart/form-data` formatting.
```

```suggestion
  For example, using the following configuration with `multipart/form-data` formatting, you can `POST` a file for transcription using the Whisper-2 transcription model:
```
Removing some repetition.